rpcs3/Emu/RSX/Program/ProgramStateCache.cpp

```diff
@@ -163,7 +174,7 @@ vertex_program_utils::vertex_program_metadata vertex_program_utils::analyse_vert
 				{
 					current_instruction = call_stack.top();
 					call_stack.pop();
-					continue;
+					jump_to_ip;
```

Comment:
just jump_to_ip; ?

Reply:
Rewrote it a bit, it was kinda janky as I wrote it half asleep. Retains the previous flow now.